thoth/messaging/adviser_trigger.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+# thoth-messaging
+# Copyright(C) 2020 Kevin Postlethwait
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This is Thoth Messaging module for AdviseJustificationMessage."""
+
+import logging
+from typing import Any
+from typing import Optional
+from typing import Dict
+
+import faust
+
+from .message_base import MessageBase
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AdviserTriggerMessage(MessageBase):
+    """Class used for Advise justification events on Kafka topic."""
```

Comment:
Adjust docstrings, also below :)

Reply:
Missing only this docstring I think @KPostOffice and this can be ready for me :)